cli/command/volume/prune_test.go

```diff
@@ -73,13 +73,15 @@ func TestVolumePruneSuccess(t *testing.T) {
 	testCases := []struct {
 		name            string
 		args            []string
+		input           string
 		volumePruneFunc func(args filters.Args) (types.VolumesPruneReport, error)
 	}{
 		{
-			name: "all",
-			args: []string{"--all"},
+			name:  "all",
+			args:  []string{"--all"},
+			input: "y",
 			volumePruneFunc: func(pruneFilter filters.Args) (types.VolumesPruneReport, error) {
-				assert.Check(t, is.Equal([]string{"true"}, pruneFilter.Get("all")))
+				assert.Check(t, is.Equal("true", pruneFilter.Get("all")[0]))
```

Comment:
Perhaps `is.DeepEqual` would work for this one, it we also want to make sure we only have a single value 🤔

I'm a bit surprised that `is.Equal` didn't complain about this (wondering if it should do so (non-comparable types))? Perhaps worth looking if we can contribute a fix for that in `gotest.tools`

```suggestion
				assert.Check(t, is.DeepEqual([]string{"true"}, pruneFilter.Get("all")))
```

Reply:
`is.Equal` does seem to complain, but only at runtime, and previously we were actually not even running that code (without running `cli.SetIn()` with an input, we were actually entering the code path where the user presses `ENTER` or `'n'/'N'` during the confirmation dialog. And yes, the output messages were the same since the prune affected 0B in either case 😅

Reply:
🤦 😂